server/src/main/java/org/elasticsearch/search/query/QuerySearchResult.java

```diff
@@ -54,7 +55,7 @@
     private TotalHits totalHits;
     private float maxScore = Float.NaN;
     private DocValueFormat[] sortValueFormats;
-    private InternalAggregations aggregations;
```

Comment:
Can you add a comment here to explain why we use a delayable writable ?

Reply:
++